docs/modules/integrations/pages/jira/create-a-jira-ticket.adoc

```diff
@@ -1,28 +1,28 @@
-= Create a Jira ticket
-:navtitle: Create a Jira ticket
-
-Learn how to create a Jira ticket directly from xref:session-explorer:index.adoc[Session Explorer].
-
-== Before you start
-
-You'll need to verify that xref:integrations:jira/enable-jira-integration.adoc[Jira has been enabled for your organization].
-
-== Create a Jira ticket
-
-In Kobiton, select *Sessions* then xref:session-explorer:search-for-a-session.adoc[search for a session].
-
-image:session-explorer:search-bar-session-criteria-context.png[width=1000,alt="Search for sessions"]
-
-Select a session.
-
-image:session-explorer:session-search-context.png[width=1000,alt="A context of Session page in Kobiton portal"]
-
-In the session overview, select *Jira ticket > Create*.
-
-image:integrations:jira-create-ticket-button.png[width="500",alt="Select Jira ticket and then Create in Session Overview"]
-
-Fill out the *Project*, *Issue Type*, and *Title*.
-
-image:integrations:jira-create-ticket-context.png[width="500",alt="Fill out required fields in the create ticket screen then select Create"]
-
-You can choose to fill out the rest of the form, or leave it as is. When you're finished, select *Create*.
+= Create a Jira ticket
+:navtitle: Create a Jira ticket
+
+Learn how to create a Jira ticket directly from xref:session-explorer:index.adoc[Session Explorer].
+
+== Before you start
+
+You'll need to verify that either xref:integrations:jira/enable-jira-integration.adoc[Jira Cloud] or xref:integrations:jira/enable-jira-data-center-integration.adoc[Jira Data Center] has been enabled for your organization.
+
+== Create a Jira ticket
+
+In Kobiton, select *Sessions* then xref:session-explorer:search-for-a-session.adoc[search for a session].
+
+image::session-explorer:search-bar-session-criteria-context.png[width=1000,alt="Search for sessions"]
+
+Select a session.
+
+image::session-explorer:session-search-context.png[width=1000,alt="A context of Session page in Kobiton portal"]
+
+From **Session Overview** or **Session Explorer**, select **Jira ticket** -> **Create**.
+
+image::integrations:jira-create-ticket-button.png[width="500",alt="Select Jira ticket and then Create in Session Overview"]
+
+Fill out the *Project*, *Issue Type*, and *Title*. A link to the session and other session details are prepopulated in the Description.

```

Comment:
If they are not signed into Jira - will they get a modal to log in?

Reply:
Yes, I have updated this point.